os/board/rtl8720e/src/component/wifi/wpa_supplicant/src/crypto/tls_polarssl.c

```diff
@@ -81,7 +81,7 @@ void my_debug(void *ctx, int level, const char *str)
 }
 
 #include <mbedtls/ssl.h>
-#include <mbedtls/net.h>
+#include <mbedtls/net_sockets.h>
 #include <mbedtls/ssl_internal.h>
```

Comment:
It should be changed to `mbedtls/ssl_misc.h`

Reply:
fixed.